### PR TITLE
i18n(es): Update `google-firebase` & `heroku`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updated `google-firebase` & `heroku` based on these commits: 
- 39720c940acc44cc19b2216106c28c75b5758b3d
- 2b5c5a36b7e43d89e446bfdf36b1360bd730b46d
- d0e247728ab7ea95f28d3b8871573e4b01ca6296

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/deploy/google-firebase.mdx [hunk 1]
```diff
@@ -4,33 +4,127 @@ description: Cómo desplegar tu proyecto de Astro usando Firebase Hosting de Goo
...
+
+Tu proyecto de Astro puede ser desplegado en Firebase como un sitio estático o como un sitio renderizado del lado del servidor (SSR).
+
+### Sitio estático
+
+Tu proyecto de Astro es un sitio estático por defecto. No necesitas ninguna configuración extra para desplegar un sitio estático de Astro en Firebase.
+
+### Adaptador para SSR
+
+To enable SSR in your Astro project and deploy on Firebase add the [Node.js adapter](/es/guides/integrations-guide/node/).
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/google-firebase.mdx:23]: Falta traducir

File: src/content/docs/es/guides/deploy/google-firebase.mdx [hunk 2]
```diff
@@ -4,33 +4,127 @@ description: Cómo desplegar tu proyecto de Astro usando Firebase Hosting de Goo
...
+### Sitio estático
+
+Tu proyecto de Astro es un sitio estático por defecto. No necesitas ninguna configuración extra para desplegar un sitio estático de Astro en Firebase.
+
+### Adaptador para SSR
+
+To enable SSR in your Astro project and deploy on Firebase add the [Node.js adapter](/es/guides/integrations-guide/node/). 
+
+:::note
+Deploying an SSR Astro site to Firebase requires the [Blaze plan](https://firebase.google.com/pricing) or higher.
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/google-firebase.mdx:26]:
```suggestion
```

File: src/content/docs/es/guides/deploy/google-firebase.mdx [hunk 3]
```diff
@@ -4,33 +4,127 @@ description: Cómo desplegar tu proyecto de Astro usando Firebase Hosting de Goo
...
 
-   ```json title="firebase.json"
-   {
-     "hosting": {
-       "public": "dist",
-       "ignore": []
-     }
-   }
-   ```
+4. Inicializa el servicio de Hosteo de Firebase en tu proyecto. Esto creará un archivo `firebase.json` y `.firebaserc` en la raíz de tu proyecto.
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/google-firebase.mdx:92]:
```suggestion
4. Inicializa el servicio de Alojamiento de Firebase en tu proyecto. Esto creará un archivo `firebase.json` y `.firebaserc` en la raíz de tu proyecto.
```

File: src/content/docs/es/guides/deploy/google-firebase.mdx [hunk 4]
```diff
@@ -4,33 +4,127 @@ description: Cómo desplegar tu proyecto de Astro usando Firebase Hosting de Goo
...
+  </PackageManagerTabs>
 
-   ```json title=".firebaserc"
-   {
-     "projects": {
-       "default": "<TU_ID_DE_FIREBASE>"
-     }
-   }
-   ```
+5. Despliega tu sitio en el servicio de Hosteo de Firebase. Esto construirá tu sitio de Astro y lo desplegará en Firebase.
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/google-firebase.mdx:112]:
```suggestion
5. Despliega tu sitio en el servicio de Alojamiento de Firebase. Esto construirá tu sitio de Astro y lo desplegará en Firebase.
```

File: src/content/docs/es/guides/deploy/heroku.mdx [hunk 5]
```diff
@@ -7,6 +7,10 @@ i18nReady: true
 
 [Heroku](https://www.heroku.com/) es una platform-as-a-service para construir, ejecutar, y administrar aplicaciones modernas en la nube. Puedes desplegar un proyecto de Astro en Heroku usando esta guía.
 
+:::danger
+Las siguientes instrucciones usan [el paquete obsoleto `heroku-static-buildpack`](https://github.com/heroku/heroku-buildpack-static#warning-heroku-buildpack-static-is-deprecated). Por favor ve la [documentación de Heroku para usar `heroku-buildpack-nginx`](https://github.com/dokku/heroku-buildpack-nginx) en su lugar.
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/heroku.mdx:11]:
```suggestion
Las siguientes instrucciones usan [el paquete obsoleto `heroku-static-buildpack`](https://github.com/heroku/heroku-buildpack-static#warning-heroku-buildpack-static-is-deprecated). Por favor consulta la [documentación de Heroku para usar `heroku-buildpack-nginx`](https://github.com/dokku/heroku-buildpack-nginx) en su lugar.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

File: src/content/docs/es/guides/deploy/google-firebase.mdx [hunk 6]
```diff
@@ -4,33 +4,127 @@ description: Cómo desplegar tu proyecto de Astro usando Firebase Hosting de Goo
...
+    ```shell
+    pnpm install firebase-tools
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn add firebase-tools
+    ```
+    </Fragment>
+  </PackageManagerTabs>
```
Comment (VoxelMC) [inline src/content/docs/es/guides/deploy/google-firebase.mdx:50]:
```suggestion
    <PackageManagerTabs>
      <Fragment slot="npm">
      ```shell
      npm install firebase-tools
      ```
      </Fragment>
      <Fragment slot="pnpm">
      ```shell
      pnpm install firebase-tools
      ```
      </Fragment>
      <Fragment slot="yarn">
      ```shell
      yarn add firebase-tools
      ```
      </Fragment>
    </PackageManagerTabs>
```

Comment (VoxelMC) [reply src/content/docs/es/guides/deploy/google-firebase.mdx:50]: These tabs should be indented more with the list, as well as the rest below
For some reason, the indentation is only detected with a tab-width of 4

File: src/content/docs/es/guides/deploy/google-firebase.mdx [hunk 7]
```diff
@@ -4,33 +4,127 @@ description: Cómo desplegar tu proyecto de Astro usando Firebase Hosting de Goo
...
+    ```shell
+    pnpm exec firebase login
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn firebase login
+    ```
+    </Fragment>
+  </PackageManagerTabs>
```
Comment (VoxelMC) [inline src/content/docs/es/guides/deploy/google-firebase.mdx:70]:
```suggestion
    <PackageManagerTabs>
      <Fragment slot="npm">
      ```shell
      npx firebase login
      ```
      </Fragment>
      <Fragment slot="pnpm">
      ```shell
      pnpm exec firebase login
      ```
      </Fragment>
      <Fragment slot="yarn">
      ```shell
      yarn firebase login
      ```
      </Fragment>
    </PackageManagerTabs>
```

Comment (VoxelMC) [reply src/content/docs/es/guides/deploy/google-firebase.mdx:70]: I won't do the rest, but I can if you want me to!

**Review by VoxelMC — COMMENTED** [review]
This one looked like a lot of work! Thank you, dreyfus :)
I left some comments as a Non-native-speaking reviewer, hope they help!

EDIT: Nevermind, ignore those! It is also like that in the English, so I will just make a PR later going over the formatting in all languages for this page.

`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

**Comment by dreyfus92** [comment]
Thank you @VoxelMC and @Waxer59 for your reviews 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Pull request merged**